Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -180,8 +199,24 @@ def __init__(self,opt,mode='init',options=''):
             "nowmIO": {},
             "Multicore" : opt.nThreads,                       # this is the per-taskchain Multicore; it's the default assigned to a task if it has no value specified 
             "EventStreams": self.numberOfStreams,
-            "KeepOutput" : False
+            "KeepOutput" : False,
+            "RequiresGPU" : None,
+            "GPUParams": None
+            }
+        self.defaultGPUParams={
+            "GPUMemoryMB": self.GPUMemoryMB,
+            "CUDACapabilities": self.CUDACapabilities,
+            "CUDARuntime": self.CUDARuntime
             }
+        
+        self.dictGPUName={"GPUName": self.GPUName}
+        if self.GPUName: self.defaultGPUParams.update(self.dictGPUName)
```

Comment:
If the `self.dictGPUName` is used only on line 213, how about
```suggestion
        if self.GPUName: self.defaultGPUParams.update({"GPUName": self.GPUName})
```
?

(same for the two other options below)

Reply:
Thx @makortel 
Fixed.